src/ensembl/src/shared/components/error-screen/ErrorScreen.scss

```diff
@@ -36,36 +30,86 @@ $generalErrorImagesTotalWidth: calc(
   flex: 0 0 auto;
 }
 
+.generalErrorTopMessage {
+  font-weight: bold;
+}
+
 .generalErrorBody {
   flex-grow: 1;
   display: flex;
   flex-direction: column;
-  justify-content: center;
+  justify-content: flex-start;
   align-items: center;
   text-align: center;
+  padding-top: 100px;
 }
 
-.generalErrorImages {
-  img {
-    width: 170px;
-    height: 170px;
-  }
+.generalErrorBottomMessage {
+  font-size: 12px;
+}
 
-  @media screen and (min-width: $generalErrorImagesTotalWidth) {
-    margin-top: 10vh;
-    margin-bottom: calc(#{$generalErrorImageGap} / 2);
+.reloadButton {
+  padding-top: 40px;
+}
 
-    img:not(:last-child) {
-      margin-right: $generalErrorImageGap;
-    }
+.moreOptions {
+  padding: 50px 0 20px 0;
+}
+.resetOption {
+  color: $red;
+  button {
+    margin-top: 24px;
   }
+}
+.generalErrorImage {
+  display: flex;
+  font-size: 12px;
 
-  @media screen and (max-width: $generalErrorImagesTotalWidth) {
-    display: inline-flex;
-    flex-direction: column;
+  .vennCircle {
+    display: flex;
+    width: 195px;
+    height: 195px;
 
-    img:not(:last-child) {
-      margin-bottom: calc(#{$generalErrorImageGap} / 2);
+    .vennCircleLeft {
+      display: flex;
+      align-items: center;
+      border-radius: 50%;
+      background-color: rgba(183, 192, 200, 0.24);
+      position: relative;
+      left: 32px;
+      text-align: right;
+
+      span {
+        margin-right: 83px;
+        margin-left: 30px;
+      }
+    }
+    .vennCircleRight {
+      display: flex;
+      align-items: center;
+      border-radius: 50%;
+      background-color: rgba(183, 192, 200, 0.24);
+      position: relative;
```

Comment:
there are still lots of CSS rules in common between `.vennCircleRight ` and `.vennCircleLeft`; they should probably go in `.vennCircle`

Reply:
Not sure about this. I have spent some time to play with it but I couldn't move any of these. May be I am wrong, have you tried it?